packages/api/endpoints/executions.js

```diff
@@ -11,10 +13,63 @@ const {
   translatePostgresExecutionToApiExecution,
 } = require('@cumulus/db');
 const Search = require('@cumulus/es-client/search').Search;
+const {
+  addToLocalES,
+  indexExecution,
+} = require('@cumulus/es-client/indexer');
 const models = require('../models');
+const { isBadRequestError } = require('../lib/errors');
 const { getGranulesForPayload } = require('../lib/granules');
+const { writeExecutionApiRecord } = require('../lib/writeRecords/write-execution');
 const { validateGranuleExecutionRequest } = require('../lib/request');
 
+const log = new Logger({ sender: '@cumulus/api/collections' });
+/**
+ * create an execution
+ *
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Promise<Object>} the promise of express response object
+ */
+async function create(req, res) {
+  const {
+    executionModel = new models.Execution(),
+    knex = await getKnexClient(),
+  } = req.testContext || {};
+
+  const execution = req.body || {};
+  const { arn } = execution;
+
+  if (!arn) {
+    return res.boom.badRequest('Field arn is missing');
+  }
+
+  if (await executionModel.exists({ arn })) {
+    return res.boom.conflict(`A record already exists for ${arn}`);
+  }
+
+  execution.updatedAt = Date.now();
+  execution.createdAt = Date.now();
+
+  try {
+    await writeExecutionApiRecord({ record: execution, knex });
+
+    if (inTestMode()) {
```

Comment:
Reminder (not for you necessarily): when we bring this work into the feature branch, we need to update this endpoint to also write directly to Elasticsearch